rd_rw612_bga/rw612/ethernet tests/net/wifi/wifi_nm build is failed

**Describe the bug**
Build of the tests/net/wifi/wifi_nm test for rd_rw612_bga/rw612/ethernet build is failed.

**To Reproduce**
Steps to reproduce the behavior:
1. west build -p -b rd_rw612_bga/rw612/ethernet tests/net/wifi/wifi_nm -T net.wifi
2. See build error:
```
> west build -p -b rd_rw612_bga/rw612/ethernet tests/net/wifi/wifi_nm -T net.wifi

-- west build: making build dir C:\_ddm\ZEPHYR\zephyr_vs\zephyr\build pristine
-- west build: generating a build system
Loading Zephyr default modules (Zephyr base).
-- Application: C:/_ddm/ZEPHYR/zephyr_vs/zephyr/tests/net/wifi/wifi_nm
-- CMake version: 3.28.1
-- Found Python3: C:/Python311/python.exe (found suitable version "3.11.4", minimum required is "3.8") found components: Interpreter
-- Cache files will be written to: C:/_ddm/ZEPHYR/zephyr_vs/zephyr/.cache
-- Zephyr version: 3.7.99 (C:/_ddm/ZEPHYR/zephyr_vs/zephyr)
-- Found west (found suitable version "1.2.0", minimum required is "0.14.0")
-- Board: rd_rw612_bga, qualifiers: rw612/ethernet
-- ZEPHYR_TOOLCHAIN_VARIANT not set, trying to locate Zephyr SDK
-- Found host-tools: zephyr 0.16.8 (C:/Users/nxa12205/zephyr-sdk-0.16.8)
-- Found toolchain: zephyr 0.16.8 (C:/Users/nxa12205/zephyr-sdk-0.16.8)
-- Found Dtc: C:/Users/nxa12205/.mcuxpressotools/dtc-msys2/tools/usr/bin/dtc.exe (found suitable version "1.6.1", minimum required is "1.4.6")
-- Found BOARD.dts: C:/_ddm/ZEPHYR/zephyr_vs/zephyr/boards/nxp/rd_rw612_bga/rd_rw612_bga_rw612_ethernet.dts
-- Generated zephyr.dts: C:/_ddm/ZEPHYR/zephyr_vs/zephyr/build/zephyr/zephyr.dts
-- Generated devicetree_generated.h: C:/_ddm/ZEPHYR/zephyr_vs/zephyr/build/zephyr/include/generated/zephyr/devicetree_generated.h
-- Including generated dts.cmake file: C:/_ddm/ZEPHYR/zephyr_vs/zephyr/build/zephyr/dts.cmake

warning: NRF_WIFI_BUILD_ONLY_MODE (defined at drivers/wifi/nrfwifi/Kconfig.nrfwifi:703) was assigned
the value 'y' but got the value 'n'. Check these unsatisfied dependencies: WIFI_NRF70 (=n). See
http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_NRF_WIFI_BUILD_ONLY_MODE and/or look up
NRF_WIFI_BUILD_ONLY_MODE in the menuconfig/guiconfig interface. The Application Development Primer,
Setting Configuration Values, and Kconfig - Tips and Best Practices sections of the manual might be
helpful too.

Parsing C:/_ddm/ZEPHYR/zephyr_vs/zephyr/tests/net/wifi/wifi_nm/Kconfig
Loaded configuration 'C:/_ddm/ZEPHYR/zephyr_vs/zephyr/boards/nxp/rd_rw612_bga/rd_rw612_bga_rw612_ethernet_defconfig'
Merged configuration 'C:/_ddm/ZEPHYR/zephyr_vs/zephyr/tests/net/wifi/wifi_nm/prj.conf'
Merged configuration 'C:/_ddm/ZEPHYR/zephyr_vs/zephyr/build/zephyr/misc/generated/extra_kconfig_options.conf'
Configuration saved to 'C:/_ddm/ZEPHYR/zephyr_vs/zephyr/build/zephyr/.config'
Kconfig header saved to 'C:/_ddm/ZEPHYR/zephyr_vs/zephyr/build/zephyr/include/generated/zephyr/autoconf.h'
-- Found GnuLd: c:/users/nxa12205/zephyr-sdk-0.16.8/arm-zephyr-eabi/arm-zephyr-eabi/bin/ld.bfd.exe (found version "2.38")
-- The C compiler identification is GNU 12.2.0
-- The CXX compiler identification is GNU 12.2.0
-- The ASM compiler identification is GNU
-- Found assembler: C:/Users/nxa12205/zephyr-sdk-0.16.8/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc.exe
Load components for RW612:
driver_common component is included.
driver_reset component is included.
device_CMSIS component is included.
CMSIS_Include_core_cm component is included.
device_startup component is included.
driver_lpc_gpio component is included.
driver_pint component is included.
driver_inputmux component is included.
driver_inputmux_connections component is included.
driver_ostimer component is included.
driver_flexcomm component is included.
driver_flexcomm_usart component is included.
driver_trng component is included.
driver_imu component is included.
driver_cache_cache64 component is included.
component_mflash_rdrw610 component is included.
component_mflash_common component is included.
driver_flexspi component is included.
component_osa_zephyr component is included from C:/_ddm/ZEPHYR/zephyr_vs/modules/hal/nxp/mcux/mcux-sdk/components/osa/set_component_osa.cmake.
driver_conn_fwloader component is included.
driver_ocotp component is included.
component_wireless_imu_adapter component is included.
driver_gdma component is included.
CMake Error at C:/_ddm/ZEPHYR/zephyr_vs/modules/hal/nxp/zephyr/src/rw61x/CMakeLists.txt:42 (message):
  Couldn't find signed firmware !
  C:/_ddm/ZEPHYR/zephyr_vs/modules/hal/nxp/zephyr/blobs/rw61x/rw61x_sb_wifi_a2.bin


-- Configuring incomplete, errors occurred!
FATAL ERROR: command exited with status 1: 'C:\Users\nxa12205\.mcuxpressotools\cmake-3.28.1-windows-x86_64\bin\cmake.EXE' -DWEST_PYTHON=C:/Python311/python.exe '-BC:\_ddm\ZEPHYR\zephyr_vs\zephyr\build' -GNinja -DBOARD=rd_rw612_bga/rw612/ethernet -DCONFIG_NRF_WIFI_BUILD_ONLY_MODE=y '-SC:\_ddm\ZEPHYR\zephyr_vs\zephyr\tests\net\wifi\wifi_nm'
```


**Expected behavior**
Successful build without error.

**Impact**
This test is part of Zephyr CI, so PR CI checks related to the rd_rw612_bga board are failed with error:

```
ERROR   - Cmake build failure: /__w/zephyr/zephyr/tests/net/wifi/wifi_nm for rd_rw612_bga/rw612/ethernet
INFO    - 301/674 rd_rw612_bga/rw612/ethernet tests/net/wifi/wifi_nm/net.wifi                     ERROR Cmake build failure (build)
INFO    - /__w/zephyr/zephyr/twister-out/rd_rw612_bga_rw612_ethernet/tests/net/wifi/wifi_nm/net.wifi/build.log

...

INFO    - The following issues were found (showing the top 10 items):
INFO    - 1) tests/net/wifi/wifi_nm/net.wifi on rd_rw612_bga/rw612/ethernet error (Cmake build failure)
INFO    - 
INFO    - To rerun the tests, call twister using the following commandline:
INFO    - west twister -p <PLATFORM> -s <TEST ID>, for example:
INFO    - 
INFO    - west twister -p rd_rw612_bga/rw612/ethernet -s tests/net/wifi/wifi_nm/net.wifi
INFO    - or with west:
INFO    - west build -p -b rd_rw612_bga/rw612/ethernet tests/net/wifi/wifi_nm -T net.wifi
INFO    - -+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
INFO    - Run completed

...

Error: Error: failed to run script step: command terminated with non-zero exit code: Error executing in Docker Container: 1
Error: Process completed with exit code 1.
Error: Executing the custom container implementation failed. Please contact your self hosted runner administrator.
```


**Environment (please complete the following information):**

 - OS: Windows
 - Toolchain Zephyr SDK
 - Commit SHA   74c5c84c66d59320ab39b58662d60e6cb6f188c2
 
**Additional context**
It is critical, as it blocks PR CI.

--- a/tests/net/wifi/wifi_nm/testcase.yaml
+++ b/tests/net/wifi/wifi_nm/testcase.yaml
@@ -8,5 +8,4 @@ tests:
       - CONFIG_NRF_WIFI_BUILD_ONLY_MODE=y
     tags: wifi
     platform_exclude:
-      - frdm_rw612 # Requires binary blobs to build
-      - rd_rw612_bga # Requires binary blobs to build
+      - rd_rw612_bga/rw612/ethernet # Requires binary blobs to build

